packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -930,6 +933,7 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
         removeLeftPadding: false,
         removeTopPadding: true,
         removeRightPadding: false,
+        removeBottomPadding: false,
```

Comment:
bah, github doesn't let me comment on unchanged lines. You'd want to remove the bottom padding from the body now no? Since it's 'consumed'.

Reply:
The body already does remove the bottom padding if necessary (line 861). Bottom padding removal is defaulted to `widget.resizeToAvoidBottomPadding` in `_addIfNonNull()`

Reply:
If in the ultimate future, resizeToAvoidBottomPadding is gone, don't we still need to explicitly set removebottompadding to true if bottom tab bar != null?

Reply:
In the future the padding is the safe area padding so we don't want to remove it, we want the nav bar to adapt itself for the additional padding.

Reply:
Done in #13437